.github/workflows/semantic-pr.yml

```diff
@@ -52,7 +52,7 @@ jobs:
     steps:
       - uses: amannn/action-semantic-pull-request@db6e259b93f286e3416eef27aaae88935d16cf2e # pin@v3.4.0
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why do we need a change in the semantic-PR workflow?

Reply:
Point 2 of the description: Ability to run actions on opened PRs.

Reply:
From the linked docs I understand that the backport workflow needs a different token than `GITHUB_TOKEN` so that the push/PR events it creates trigger the test suite.

I don't understand why the semantic-PR workflow needs to create events that trigger workflow runs. Which events are those? And which workflows?

Reply:
The required change seemed to be already in the original PR.